tests/test_unit.py

```diff
@@ -4433,38 +4291,37 @@ def key_parse_action(toks):
         success, _ = parser.runTests([sample2])
         self.assertTrue(success, "Failed indentedBlock multi-block test for issue #87")
 
-class IndentedBlockScanTest(ParseTestCase):
-    def get_parser(self):
-        """
-        A valid statement is the word "block:", followed by an indent, followed by the letter A only, or another block
-        """
-        stack = [1]
-        block = pp.Forward()
-        body = pp.indentedBlock(pp.Literal('A') ^ block, indentStack=stack, indent=True)
-        block <<= pp.Literal('block:') + body
-        return block
+    def testIndentedBlockScan(self):
+        def get_parser():
```

Comment:
This could be moved to the bottom of the file, but I thought keeping it local to the tests that use it would be good.  So it is an inner method.